lib/voyager_helpers/liberator.rb

```diff
@@ -563,7 +563,11 @@ def valid_ascii(string)
       end
 
       def valid_codepoints(string)
-        string.codepoints.map{|c| c.chr(Encoding::UTF_8)}.join
+        unless string.nil?
+          string.codepoints.map{|c| c.chr(Encoding::UTF_8)}.join
```

Comment:
the unless/else pattern is unnecessarily verbose in this case. The one-liner below is equivalent. Both functions will return nil when string is nil.
```
string.codepoints.map{|c| c.chr(Encoding::UTF_8)}.join unless string.nil?
```